src/labone/node_info.py

```diff
@@ -72,11 +72,44 @@ class NodeInfo:
     def __init__(self, info: NodeInfoType):
         self._info: NodeInfoType = info
 
+    @classmethod
+    def plain_default_info(cls, path: str | None = None) -> NodeInfoType:
+        """Get default info for a node.
+
+        Returns:
+            Default info for a node.
+        """
+        info: NodeInfoType = {
+            "Description": "",
+            "Properties": "Read, Write, Setting",
```

Comment:
```suggestion
            "Properties": "Read, Write",
```
issue: `Settings` should not be used by default.